README.md

```diff
@@ -1,30 +1,27 @@
-
-Next there is the recommendation version, one can click on tags (at least one is mandatory) on the genre of the book, this would allow that upon a good opinion of a book, the app will recommend using an AI the closest books that match the genres you like. This would of course use geolocation as it must match you with books that lie somewhere in your near vicinity. 
-
-There is also an AI that takes in a request that can be somewhat vague and come up with better books according to your current mood and desires.
-
-Lastly, there is a method of connecting with other people to propose the book exchange, or maybe something further… Each account has a few things, mostly based on the google login, which would allow you to enter a chatbox with the people you want a book from, it would be a very simple chatbox, with minimal memory of previous texts, but long enough to set up a meeting and hopefully provide you with a new book and perhaps a new close acquaintance.
-
-- For the split app model, we will use the public cloud services not only for the google login, but also for storing the repository of books and the chatbox.
-- A google login will be used to share info between devices.
-- It will use geolocation for the recommendation and camera for easier creation of book entries
-- The scans of the books and entry creation and managing will be offline and doable without connectivity, it will also store a state so you can see your directory as it was the last time the android was connected.It will update the cloud upon reconnecting.
-
-## Links
-[Figma link](https://www.figma.com/design/uyHS0PV5RBnmToK2JuTb4h/Untitled?node-id=0-1&m=dev&t=VdHNM8v7U5bhvsW7-1)
-
-[Team review sheet](https://docs.google.com/spreadsheets/d/1hyi3ISzlwAXZ63ztd7Q84RkBFf24cPgatv3uUx4Y4hc/edit?usp=sharing)
-
-[Team minutes](https://docs.google.com/spreadsheets/d/1YF34CWge25Y1hSQsP5UWHCDoyYNJaxIPasSb13gWYao/edit?usp=sharing)
-
-[Architecture Diagram](https://lucid.app/lucidchart/a07f4bb7-a054-40e9-95be-d4cbb0485822/edit?viewport_loc=-1324%2C33%2C6059%2C2938%2C0_0&invitationId=inv_d502a57d-8d89-4e37-9a4f-f06ec2cf1941)
+## Introduction
+Welcome to BookSwap. The application consists of an online version of the free book exchange boxes often found in small villages scattered throughout the whole of the Swiss countryside, it helps avoid having to search for said box exchanges and additionally allows to search for a more desirable book than what perhaps would be in the box if one were to simply walk up to said box and have a look.
+
+## Features
+- **Adding Books:**  BookSwap offers three user-friendly methods to add books to your account. The first is the traditional approach, where you manually enter the book details. Alternatively, you can simply input the ISBN for quick addition. Lastly, BookSwap leverages an advanced AI model to extract book information automatically from a photo of the book, making the process seamless and efficient.
```

Comment:
Here we have to make sure that it says the picture has to be taken of the back cover, the front cover will simply not work